packages/vuetify/src/components/VIcon/VIcon.ts

```diff
@@ -153,19 +153,25 @@ const VIcon = mixins(
       return h(this.hasClickListener ? 'button' : this.tag, data, newChildren)
     },
     renderSvgIcon (icon: string, h: CreateElement): VNode {
-      const fontSize = this.getSize()
       const svgData: VNodeData = {
         class: 'v-icon__svg',
         attrs: {
           xmlns: 'http://www.w3.org/2000/svg',
           viewBox: '0 0 24 24',
-          height: fontSize || '24',
-          width: fontSize || '24',
           role: 'img',
           'aria-hidden': true,
         },
       }
 
+      const size = this.getSize()
```

Comment:
```suggestion
      const size = this.getSize()

```

Reply:
Why? That's how it is everywhere else and in the original code.